client/my-sites/plans/current-plan/current-plan-thank-you/thank-you.js

```diff
@@ -82,6 +83,21 @@ export class ThankYouCard extends Component {
 							</a>
 						</p>
 					) }
+					{ showSearchRedirects && (
+						<p className="current-plan-thank-you__followup">
+							<Button
+								primary
+								compact={ true }
+								href="customize.php?autofocus[section]=jetpack_search"
```

Comment:
Just a quick note here that this link doesn't work yet.

Reply:
yes, it is still in the making